ompi/mca/io/ompio/io_ompio_file_read.c

```diff
@@ -362,13 +362,22 @@ int mca_io_ompio_file_read_all_begin (ompi_file_t *fh,
 
     data = (mca_common_ompio_data_t *) fh->f_io_selected_data;
     fp = &data->ompio_fh;
-    if ( true == fp->f_split_coll_in_use ) {
-	printf("Only one split collective I/O operation allowed per file handle at any given point in time!\n");
-	return MPI_ERR_OTHER;
+    if (true == fp->f_split_coll_in_use) {
+        printf("Only one split collective I/O operation allowed per file handle at any given point in time!\n");
```

Comment:
These printfs should probably be fixed, but not as part of this PR.

Reply:
do you mean converting it to opal_output statements?